test/framework/easyconfigs/yeb/python-2.7.10.yeb

```diff
@@ -0,0 +1,132 @@
+_internal_variables_:
+    - &numpyversion 1.9.2
+    - &scipyversion 0.15.1
+    - &name Python
+    - &version 2.7.10
+
+easyblock: ConfigureMake
+
+name: Python
+version: 2.7.10
```

Comment:
use `*version` here too, or don't include `version` in the list above; same for `name`